src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const centralFrame = document.querySelector('.gallery__large-img');
+const list = document.querySelector('.gallery__list');
+
+list.addEventListener('click', e => {
+  const item = e.target.closest('.list-item__link');
+
+  if (item) {
```

Comment:
Move the assignment of the centralFrame.src inside the if statement to avoid errors when item is not valid